app/javascript/components/clickable-dropdown/index.vue

```diff
@@ -7,14 +7,23 @@
     </div>
     <div class="select__body" slot="body">
       <div v-if="bodyTitle" class="select-body__title">{{ bodyTitle }}</div>
-      <div
-        v-for="(item, index) in items"
-        class="select__option"
-        @click="itemClicked({ item, index })"
-        :key="index"
-      >
-        {{ item.name }}
-      </div>
+      <dropdown v-for="(item, org_index) in items" v-if="item.login" class="select">
```

Comment:
Custom elements in iteration require 'v-bind:key' directives     vue/valid-v-for